src/nwb_datajoint/common/common_curation.py

```diff
@@ -160,11 +173,11 @@ def _sorting_after_reject(sorting, quality_metrics, reject_params):
         isi_violation_frac_threshold = reject_params['isi_violation_frac_threshold']
         
         # Remove units that had too many isi violations. Keep the rest
-        passed_units_ids = [unit_id for unit_id in sorting.get_unit_ids() if isi_violation_fractions[unit_id] < isi_violation_rate_threshold]
+        isi_passed_units_ids = [unit_id for unit_id in nn_noise_overlap_passed_sorting.get_unit_ids() if isi_violation_fractions[unit_id] < isi_violation_rate_threshold]
         
-        isi_passed_sorting = sorting.select_units(passed_units_ids)
+        isi_noise_overlap_passed_sorting = nn_noise_overlap_passed_sorting.select_units(isi_passed_units_ids)
         
-        return isi_passed_sorting
+        return isi_noise_overlap_passed_sorting
```

Comment:
this should probably called something more precise, e.g. `noise_rejected_sorting`

Reply:
Mind if I call it `noise_passed_sorting`? It makes more sense to me. It would read as "a version of the sorting that has passed noise checks."

Reply:
yes